recipes/units/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+import os
+from conans import ConanFile, tools
+
+
+class UnitsConan(ConanFile):
+    name = "units"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nholthaus/units"
+    description = "A compile-time, header-only, dimensional analysis and unit conversion library built on c++14 with no dependencies"
+    topics = ("unit-conversion", "dimensional-analysis", "cpp14",
+              "template-metaprogramming", "compile-time", "header-only",
+              "no-dependencies")
+    license = "MIT"
+    no_copy_source = True
+
+    _source_subfolder = "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy("units.h",
+                  dst="include/units",
```

Comment:
The cmake script does not copy the headers in a `units` subdirectory.
https://github.com/nholthaus/units/blob/9f09abb32641fdf8dbb2c49ee81717d434624218/CMakeLists.txt#L230-L232
```suggestion
                  dst="include",
```

Reply:
Thanks. I took the subdirectory out.